earth_enterprise/src/server/wsgi/common/utils.py

```diff
@@ -216,6 +216,7 @@ def GetApacheSchemeHostPortFromListen():
     assert port
     if not scheme:
       scheme = "https" if port == "443" else "http"
+    host = None
```

Comment:
This is about the fact that utils.py parses the apache config file itself directly, and will use the ip address of an apache bind as the "host name" for other opengee operations if an ip address is used to bind the apache listen to a specific interface.  If something like "Listen 80" is used, then it just returns the default system hostname.  If you have a bind like "Listen 127.0.0.1:80", then it actually makes 127.0.0.1 a hostname in other operations, such as the list of hosts returned by geserveradmin --listvhs.  My own impression is that this is not desired behavior, and this change makes it always fall thru and use the system hostname regardless of what was in the apache Listen directive.

Reply:
It seems this function is doing exactly what it was documented to do and what its name implies that it will do.  Is the bug with this function or is the bug really with the calling code for incorrectly using the host returned from this function?  If we are always going to return `None` for host then maybe rename this function and change the documentation to state it doesn't return a host and just have it return a tuple with two values (scheme and port) instead of three.